src/main/scala/it/pagopa/interop/notifier/service/impl/EventIdRetriever.scala

```diff
@@ -40,7 +37,7 @@ final class EventIdRetriever(
     val commander: EntityRef[Command] =
       sharding.entityRefFor(OrganizationNotificationEventIdBehavior.TypeKey, getShard(organizationId))
 
-    commander.ask(ref => UpdateOrganizationNotificationEventId(organizationId, ref)).flatMap {
+    commander.ask(ref => UpdateOrganizationNotificationEventId(organizationId, ref))(300.seconds).flatMap {
```

Comment:
I may agree on removing the package object, but hardcoding the timeout isn't a bit dangerous in terms of maintenance? shouldn't we put it in a variable?